src/components/CustomChannelComponent.tsx

```diff
@@ -109,13 +112,13 @@ export function CustomChannelComponent(props: CustomChannelComponentProps) {
               {allMessages &&
                 allMessages.length > 1 &&
                 lastMessage.sender.userId === botUser.userId &&
-                !messageMeta?.stream &&
-                isSpecialMessage(
+                !lastMessageMeta?.stream &&
+                !isSpecialMessage(
```

Comment:
Should be this https://github.com/sendbird/ai-bot-url-webdemo/blob/preprod/src/components/CustomChannelComponent.tsx#L110